02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,78 @@
+body{
+  background-color: #FAEBD7;
+}
+
+
+  header{
+text-align: center;
+}
+h1{
+  color: blue
+  font-family: "Times New Roman", Times, serif;
+}
+
+
+ol{
+  font-family: "Times New Roman", Times, serif;
+}
+
+p{
+  font-family: "Times New Roman", Times, serif;
+}
+
+section{
+  float: left;
+}
+
+aside{
+  float: right;
+  padding: 30px;
+}
+a{
+  text-decoration: none;
+}
+h2{
+  padding-bottom: 20px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+ul li{
+  margin-left: 20px;
+  padding-top: 20px;
+  padding-left: 10px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+table{
+  float: left;
+  margin-left:auto;
+  margin-right: auto;
+}
+
+th{
+  padding: 20px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+p.table{
```

Comment:
Eddie,

 

From my readings and through practical trying and error the. Notation represents a class notation.  In the case you pointed out I have a paragraph with a table class that I wanted to center.  There may be other ways to do this but this one worked for me.  Are my thoughts incorrect?

 

Steve

 

From: Eddie Garcia [mailto:notifications@github.com] 
Sent: Tuesday, August 2, 2016 1:53 PM
To: AustinCodingAcademy/intro-workbook <intro-workbook@noreply.github.com>
Cc: Steve Anderson <anderson-steven@att.net>; Author <author@noreply.github.com>
Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.Intro.2.Anderson.Steve (#84)

 

In 02LessonTwo/homework/css/style.css <https://github.com/AustinCodingAcademy/intro-workbook/pull/84#discussion_r73215232> :

> +  padding-left: 10px;
> +  font-family: "Times New Roman", Times, serif;
> +}
> +
> +table{
> +  float: left;
> +  margin-left:auto;
> +  margin-right: auto;
> +}
> +
> +th{
> +  padding: 20px;
> +  font-family: "Times New Roman", Times, serif;
> +}
> +
> +p.table{

This is a great effort. Clearly you designed something great here. Question: What is this dot notation supposed to do? 

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/AustinCodingAcademy/intro-workbook/pull/84/files/b6b59d0594f9d7def775bf5777865638c099138b..b76e2ab80dec1969e2550ed8c5f8ecec786bcfde#r73215232> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ATJFMHIIjmCKgMaZwf1HY8oTH2lVxczwks5qb5IIgaJpZM4JVvJX> .  <https://github.com/notifications/beacon/ATJFMHzvuCfOCgAJP6k0NUlTmvUmP75xks5qb5IIgaJpZM4JVvJX.gif>